src/client/pages/Embark/ErrorModal.tsx

```diff
@@ -69,9 +69,11 @@ export const SetupFailedModal = ({
   return (
     <ErrorModal {...props} dynamicHeight={true} onClose={goToLandingPage}>
       <ErrorHeading>{textKeys.GENERIC_ERROR_HEADING()}</ErrorHeading>
-      <ReactMarkdown source={textKeys.GENERIC_ERROR_TEXT()} />
+      <ErrorText>
+        <ReactMarkdown source={textKeys.GENERIC_ERROR_TEXT()} />
+      </ErrorText>
       <ButtonContainer>
-        <Button onClick={onRetry}>
+        <Button onClick={() => onRetry()}>
```

Comment:
This should work just like before, no? Just a nit!

Reply:
Actually it does not work. I am not sure what the issue is exactly but this is the only way to fix it